third_party/blink/renderer/modules/ml/webnn/ml_graph_builder.cc

```diff
@@ -1712,10 +1712,8 @@ MLOperand* MLGraphBuilder::resample2d(const MLOperand* input,
     output_shape[axes[1]] = options->sizes()[1];
 
     // Compute the scales from the new shape.
-    for (wtf_size_t i = 0; i < input_rank; ++i)
-    {
-      scales[i] = static_cast<float>(output_shape[i]) / input_shape[i];
-    }
+    scales[0] = output_shape[axes[0]] / input_shape[axes[0]];
```

Comment:
Is the `static_cast<float>` missed? Otherwise, the division result would be an integer.

Fix it (and the mode) in PR https://github.com/fdwr/chromium-src-webnn-dml/pull/9

Reply:
> Is the `static_cast<float>` missed? Otherwise, the division result would be an integer.
> 
> Fix it (and the mode) in PR #9

Thanks! @huningxin